resources/views/admin/users/word_filters.twig

```diff
@@ -0,0 +1,37 @@
+{% extends "layouts.admin" %}
+{% block title %}{% endblock %}
+{% block contents %}
+    <div class="admin-body__header">
+        <h1 class="admin-body__header__title">{{ trans('admin::word_filters.title') }}</h1>
+        <div class="admin-body__header__buttons">
+            <a href="{{ url_route('admin.word_filters.add') }}" class="button"><i class="fa fa-plus"></i> {{ trans('admin::word_filters.add_button') }}</a>
+        </div>
+    </div>
+    <div class="admin-body__content" style="display: flex;flex-direction: row;">
+        <table class="admin-table admin-table--bordered users" style="width:79%;">
```

Comment:
Paging @Eric-Jackson 😛

Reply:
I mentioned to @Azareal in Discord, we will want to avoid inline style and tables. For now the classes already available on these elements should work fine for removing that inline stuff, the tables can be replaced later so I wouldn't worry about them at this moment. The class "users" should be replaced with something more appropriate `word-filters`.